docs/module_docs/filter_isotopegrouper/isotope_grouper.md

```diff
@@ -0,0 +1,27 @@
+# Isotope grouper (filter)
+
+The **Isotope grouper** is found under **Feature list methods → Isotopes → Isotope grouper (filter).**
+
+![](isotope_grouper.png)

```

Comment:
Can you use a screenshot with a retention time tolerance other than 0? maybe a few seconds or so?